src/data/contributors/ritiksoni00.json

```diff
@@ -0,0 +1,9 @@
+
+{
+    "name": "Ritik Soni",
+    "skills" : ["Nan"],
```

Comment:
Though "Nan" shows some of your skills,If you don't want to mention any skill leave it  as an empty array like `[]`